lib/rust/parser/debug/src/bin/lexer.rs

```diff
@@ -1,9 +1,5 @@
 //! Run the lexer from the command line, for understanding the early stages of the parser.
 
-// === Features ===
-#![feature(exact_size_is_empty)]
-#![feature(let_chains)]
-#![feature(if_let_guard)]
```

Comment:
Does removing some of these feature have an impact on the size of generated `.so` binary?

Reply:
Some of the switches might be a little less optimized, but it won't have a noticeable impact.